DMX.Core.Api/Controllers/LabsController.cs

```diff
@@ -8,9 +8,7 @@
 using DMX.Core.Api.Models.Foundations.Labs.Exceptions;
 using DMX.Core.Api.Models.Orchestrations.Labs.Exceptions;
 using DMX.Core.Api.Services.Orchestrations;
-using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Identity.Web.Resource;
 using RESTFulSense.Controllers;
```

Comment:
We still need the `Microsoft.AspNetCore.Authorization` and the `Microsoft.Identity.Web.Resource` - the IDE will show these as unused because we are regionalizing the `[Authorize]` decoration. So Visual Studio thinks that these are not used, but as soon as you remove the `#if` and `#endif` these directives will become active immediately.